src/util/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const isFunction = (obj: any) => {
```

Comment:
I used it for unregister event handler by index,  so I need to know if the parameter is a function or a number, but I think it is dangerous because the index when you register the event handler could change. So I remove it.
But I think a function to know if a variable is a function is needed, so I keep it here.